test/bats/callback.bats

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bats
+
+load "helpers/setup-and-teardown"
+load "helpers/ln"
+
+setup_file() {
+  clear_cache
+
+  bitcoind_init
+  start_trigger
+  start_ws_server
+  start_server
+  start_exporter
+  start_callback
+
+  lnds_init
+}
+
+teardown_file() {
+  stop_trigger
+  stop_server
+  stop_ws_server
+  stop_exporter
+  stop_subscriber
+  stop_callback
+}
+
+setup() {
+  reset_redis
+}
+
+teardown() {
+  if [[ "$(balance_for_check)" != 0 ]]; then
+    fail "Error: balance_for_check failed"
+  fi
+
+}
+
+btc_amount=1000
+usd_amount=50
+
+@test "callback endpoints" {
```

Comment:
Can't quite see - does this assert that the callback gets called?

Reply:
you mean the 

```
btc_amount=1000
usd_amount=50
```

this has been removed